upgrading/extending-daml-models/2_new_package.md

```diff
@@ -0,0 +1,91 @@
+In this step, you'll create a new package depending on `create-daml-app` from scratch. We will be
+implementing the back-end for a discussion forum. The users will be identified by their user data
+from the mini social network of the Getting Started Guide. Thus we don't need to reimplement the
+user roles.
+
+In the terminal, change directory to your home directory and create a new empty DAML project:
+
+```
+cd
+daml new forum
+cd forum
+```{{execute T1}}
+
+The first thing you need to do is to add the `create-daml-app` to the dependencies in the
+`daml.yaml`{{open}} file. After, the file will look like this:
+
+<pre class="file" data-target="clipboard">
+sdk-version: 1.1.1
+name: forum
+source: daml
+parties:
+  - Alice
+  - Bob
+version: 0.0.1
+dependencies:
+  - daml-prim
+  - daml-stdlib
+  - daml-script
+  - ../create-daml-app/.daml/dist/create-daml-app-0.1.0.dar
+sandbox-options:
+  - --wall-clock-time
+</pre>
+
+A package is simply added by pointing to file location of its `dar` (**D**AML **ar**chive) archive.
+When you build a `dar` package with `daml build` within a project without an additional output
+argument, the `dar` will be under the path `.daml/dist/` relativ to your project root.
+
+Remove the two files `daml/Main.daml` and `daml/Setup.daml`. These were added as a starting point,
+but we have our own ideas.
+
+```
+rm daml/Main.daml
+rm daml/Setup.daml
+```{{execute T1}}
```

Comment:
Same comment as before 😢These files are not visible in the IDE

Reply:
that seems to work for me, because that's a shell command independent of the IDE.

Reply:
Sorry for not making it clear @robin-da, my bad. What I meant is that the new "Forum" project and its files are not visible in the IDE. The first thing I did after creating the project was to check what's in it in the IDE (wanted to look at the code). But nothing is there as the project is not attached to it, so the two files that I'm removing are "not visible" to me. Does it make more sense now?